libs/components/a11y/src/lib/modules/skip-link/skip-link-host.component.scss

```diff
@@ -1,19 +1,31 @@
+@use 'libs/components/theme/src/lib/styles/variables' as *;
+
+.sky-skip-link {
+  --sky-skip-link-hidden-left-position: -999px;
```

Comment:
I'm not seeing any advantage to using CSS custom properties in this instance since each one is only ever referenced once. Custom properties are most useful when implementing theming where each theme defines a different value for the same rule. See the [alert component](https://github.com/blackbaud/skyux/blob/main/libs/components/indicators/src/lib/modules/alert/alert.component.scss) for examples.